### DISCUSSION TITLE
Layer Sources

### DISCUSSION THREAD

**Discussion by cnilsen** [Geosyntec/StormPiper#191] [General]
# Tile Layers 

| Layer Name | url  | link to View | 
|:--|:--|:--|
|  Imperviousness |  https://storage.googleapis.com/live_data_layers/tiles/Imperviousness/{z}/{x}/{y} |  https://storage.googleapis.com/live_data_layers/tiles/Imperviousness/index.html |  
|  Land Cover |  https://storage.googleapis.com/live_data_layers/tiles/Land_Cover/{z}/{x}/{y} |  https://storage.googleapis.com/live_data_layers/tiles/Land_Cover/index.html |  
|  Runoff |  https://storage.googleapis.com/live_data_layers/tiles/Runoff/{z}/{x}/{y} |  https://storage.googleapis.com/live_data_layers/tiles/Runoff/index.html |  
|  Total Copper  |  https://storage.googleapis.com/live_data_layers/tiles/Total_Copper/{z}/{x}/{y} |  https://storage.googleapis.com/live_data_layers/tiles/Total_Copper/index.html |  
|  Total Nitrogen |  https://storage.googleapis.com/live_data_layers/tiles/Total_Kjeldahl_Nitrogen/{z}/{x}/{y} |  https://storage.googleapis.com/live_data_layers/tiles/Total_Kjeldahl_Nitrogen/index.html |  
|  Total Suspended Solids |  https://storage.googleapis.com/live_data_layers/tiles/Total_Suspended_Solids/{z}/{x}/{y} |  https://storage.googleapis.com/live_data_layers/tiles/Total_Suspended_Solids/index.html |  
|  Total Zinc |  https://storage.googleapis.com/live_data_layers/tiles/Total_Zinc/{z}/{x}/{y} |  https://storage.googleapis.com/live_data_layers/tiles/Total_Zinc/index.html |  
|  Contours |  https://storage.googleapis.com/live_data_layers/tiles/usgs_contours/{z}/{x}/{y} |  https://storage.googleapis.com/live_data_layers/tiles/usgs_contours/index.html |  
|  Terrain |  https://storage.googleapis.com/live_data_layers/tiles/topo_shade_with_roofs/{z}/{x}/{y} |  https://storage.googleapis.com/live_data_layers/tiles/topo_shade_with_roofs/index.html |

# Vector Layers
These layers are available from the City ArcGIS Server. Use the following base url for all layers: 
`https://gis.cityoftacoma.org/arcgis/rest/services/`

### Conveyances 
| Layer Name | path  |
|:--|:--|
|Surfacewater Main |`ES/SurfacewaterNetwork/MapServer/31`\
|  Surfacewater CB Lead | `ES/SurfacewaterNetwork/MapServer/30`|  
|  Surfacewater Trunk Main |  `ES/SurfacewaterNetwork/MapServer/32`|  

### Structures 
| Layer Name | path  |
|:--|:--|
|  Surfacewater Inlet |  `ES/SurfacewaterNetwork/MapServer/25` |  
|  Surfacewater Manhole |  `ES/SurfacewaterNetwork/MapServer/23`|  
|  Surfacewater Discharge Point | `ES/SurfacewaterNetwork/MapServer/24`|
## Used in Reporting or Prioritization 
| Layer Name | path  |
|:--|:--|
|Land Use |`General/LandUseDesignations/MapServer/0`|
|Pavement Condition Index |`DART/DARTinfra_PUBLIC/MapServer/3`|
|Biodiversity Corridors |`DART/DARTenvironmental/MapServer/5`|
|Urban Heat| `ES/UrbanHeatIslandIndex/MapServer/0` |

**Comment by cnilsen** [comment]
## Color ramps for raster images 
### Continuous Base Layers 
Imperviousness 
![Imperviousness](https://github.com/Geosyntec/StormPiper/assets/4753248/100da8ee-e0d2-49eb-8708-bf170b6696f7) (0-100%) 
Runoff 
![runoff](https://github.com/Geosyntec/StormPiper/assets/4753248/6054286d-3520-4bf6-91b2-e168ee74d28f) (25 - 1016 mm) 

### Pollutants 
Note: Pollutants are shown log scaled. Ranges shown are (low, midpoint, max) on the color ramp. 

Total   Copper
![cocs](https://github.com/Geosyntec/StormPiper/assets/4753248/1fdb1738-9adf-4ae6-bd03-dda6aebf5d97)  log scale: (0, 5.6, 31 µg/L) 
Total   Nitrogen
![cocs](https://github.com/Geosyntec/StormPiper/assets/4753248/1fdb1738-9adf-4ae6-bd03-dda6aebf5d97) log scale: (0, 28.8, 829 µg/L) 
Total   Suspended Solids
![cocs](https://github.com/Geosyntec/StormPiper/assets/4753248/1fdb1738-9adf-4ae6-bd03-dda6aebf5d97) log scale: (0, 6.1, 36 mg/L) 
Total   Zinc
![cocs](https://github.com/Geosyntec/StormPiper/assets/4753248/1fdb1738-9adf-4ae6-bd03-dda6aebf5d97) log scale:  (0, 11.6, 134 µg/L) 

## Discrete Layers 

Land Cover 


Label | Raster value | Colors
-- | -- | --
No data | 0 | #ffffff
Grass/Low Vegetation | 1 | #3ead63
Shrub/Medium Vegetation | 2 | #96ca6e
Trees/Forest | 3 | #26532b
Bare soil | 4 | #a39171
Water | 5 | #476b9d
Impervious | 6 | #adacb5
Impervious - Roofs | 7 | #d8d5db

**Reply by austinorr** [reply]
I think we will just take screen shots of the tnc heat map and display them. Seems easier than reconstructing these properly.
